queries/imma-artists.sparql

```diff
@@ -3,18 +3,15 @@ prefix xhtml: <http://www.w3.org/1999/xhtml#>
 PREFIX xyz:  <http://sparql.xyz/facade-x/data/>
 PREFIX fx:   <http://sparql.xyz/facade-x/ns/>
 
-select distinct ?artistUrl ?artistNickname
+SELECT DISTINCT ?artistUrl ?artistNickname
 WHERE {
-    SERVICE <x-sparql-anything:media-type=text/html,html.selector=#az-group,location=https://imma.ie/artists/> {
+    SERVICE <x-sparql-anything:media-type=text/html,html.selector=#	,location=https://imma.ie/artists/> {
```

Comment:
The selector should ensure only links in scope are matched, why remove it?

Reply:
Sorry, it was a typo. I'm going to revert this change.